### PR TITLE
COMMERCE-1036 fixed cards height on comparison table

### PR DESCRIPTION
**Pull request by fabio-d-m**
Please review this pr after: https://github.com/liferay/com-liferay-commerce/pull/1686

### CODE REVIEW

**Comment by fabio-d-m** [comment]
@marco-leo fixed

File: commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce_compare_table.scss [hunk 1]
```diff
@@ -18,24 +18,51 @@
...
+		&:last-child {
+			border-right: none;
+		}
+	}
 
 	&__separator {
 		font-weight: 800;
 	}
-
+
```
Comment (ethanbustad) [inline commerce-theme-minium/commerce-theme-minium/src/css/minium/components/commerce_compare_table.scss:59]: errant whitespace

File: commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_commerce_compare_table.scss [hunk 2]
```diff
@@ -13,16 +13,41 @@
...
+			box-shadow: none;
+		}
+
+		&__actions {
+			bottom: 20px;
+			left: 20px;
+			position: absolute;
+			right: 20px;
+		}
+	}
```
Comment (ethanbustad) [inline commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_commerce_compare_table.scss:31]: Whitespace issues here. Indentation doesn't match surroundings, and missing a space in between `.minium-card` and `{`

File: commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_commerce_compare_table.scss [hunk 3]
```diff
@@ -13,16 +13,41 @@
...
+		border-radius: 8px;
+		border-right: 2px solid $color-bg-alt;
+		padding: 0;
+		position: relative;
+		vertical-align: top;
+
+		&:last-child {
+			border-right: none;
+		}
+	}
```
Comment (ethanbustad) [inline commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_commerce_compare_table.scss:49]: same indentation issue here

**Pull request closed without merging**